zerver/webhooks/airbrake/view.py

```diff
@@ -13,16 +13,17 @@
 
 @api_key_only_webhook_view('Airbrake')
 @has_request_variables
-def api_airbrake_webhook(request, user_profile, client, payload=REQ(argument_type='body'),
+def api_airbrake_webhook(request, user_profile,
+                         payload=REQ(argument_type='body'),
                          stream=REQ(default='airbrake')):
-    # type: (HttpRequest, UserProfile, Client, Dict[str, Any], Text) -> HttpResponse
+    # type: (HttpRequest, UserProfile, Dict[str, Any], Text) -> HttpResponse
     try:
         subject = get_subject(payload)
         body = get_body(payload)
     except KeyError as e:
         return json_error(_("Missing key {} in JSON").format(str(e)))
 
-    check_send_message(user_profile, client, 'stream', [stream], subject, body)
+    check_send_message(user_profile, request.client, 'stream', [stream], subject, body)
```

Comment:
I think we we can remove the `Client` import in all of these integrations with this change.